includes/definitions/weos.yaml

```diff
@@ -0,0 +1,16 @@
+os: weos
+text: 'Westermo WeOS'
+type: network
+icon: westermo
+group: westermo
+ifname: true
+mib_dir: westermo
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.16177
+good_if:
+    - 1/1
```

Comment:
Why did you explicitely add these good_if definitions ? they should be discovered by default as far as I see.

Reply:
> Why did you explicitely add these good_if definitions ? they should be discovered by default as far as I see. Did you test and forget to remove ?

I added these because port 1/1 and 1/2 are the SHDSL ports, which by default are ignored by LibreNMS. I tried using `good_ifType: shdsl` as well, but it doesn't seem to work. Is it not possible to specify the ifType instead?

Do you think it's better if I remove the `good_if` definitions, and instead let the users explicitly enable `ifType: shdsl` globally or per OS if needed?

Reply:
Hi @PipoCanaja, did you have time to look at this?

Reply:
Hi,
Sorry for not getting back to you earlier. This got completely out of my mind. 
Basically I don't see why shdsl ports should be skipped. I would suggest to get them discovered by default (and then, if some kind of device requires it, it can be disabled specifically...).

Reply:
Hi, no worries!
I agree with you, or is there a risk it could cause issues for existing users if SHDSL-ports are suddenly discovered?
However, I assume most users don't have any equipment with SHDSL-ports anyway.

If they should be discovered by default, will you make the necessary configuration changes separately or should I include that in this PR?

https://github.com/librenms/librenms/blob/d7c31e0ae366749ab4648668f7ac01eb7eb6ff3d/misc/config_definitions.json#L913

Reply:
Hi @PipoCanaja, sorry to distrub you again! Perhaps I should've tagged you right away in my previous comment :)
What's your thought about the above?

Reply:
You can do it in this PR indeed. We'll see if it breaks tests data or not.

Reply:
@PipoCanaja Ok. I've pushed the changes now and everything looks good from here. I captured new test data and it's identical to what it was before. Thank you.